SalesforceSDK/Salesforce.SDK.Hybrid.SmartStore/Source/Store/SmartStore.cs

```diff
@@ -165,6 +166,16 @@ public object Upsert(string soupName, string soupElt, string externalIdPath, boo
             return NativeSmartStore.Upsert(soupName, JObject.Parse(soupElt), externalIdPath, handleTx);
         }
 
+        public bool BeginDatabaseTransaction()
```

Comment:
You can get to this method via methods in Native smartstore, do we really need to expose this?